### PR TITLE
add pid.available for evict-head on linux

### PR DESCRIPTION
**Pull request by Cweiping**
Fix Issue: https://github.com/kubernetes/kubernetes/issues/98206
Reference PR: https://github.com/kubernetes/kubernetes/pull/98207

/langage en

### CODE REVIEW

**Comment by Cweiping** [comment]
/hold
Waiting for kubernetes/kubernetes#98207 marge.

**Review by ehashman — APPROVED** [review]
/sig node
/lgtm

from node perspective, once above PR merges
/cc @SergeyKanzhelev

**Comment by celestehorgan** [comment]
/approve

@Cweiping You can remove the hold yourself when your fix PR goes through using `hold cancel`.

**Comment by reylejano** [comment]
@Cweiping, I'm the release docs lead for 1.21. Is this k/website PR targeted for the `dev-1.21` branch related to a k/enhancement issue that is tracked for the v1.21 milestone?

File: content/en/docs/tasks/administer-cluster/out-of-resource.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/out-of-resource.md:124]: It might be worth emphasizing that this is compared against `RLIMIT_NPROC`. I'm suggesting this because you might think it was compared against `kernel.pid_max` or against the ceiling from the relevant `pids` cgroup controller.

Comment (SergeyKanzhelev) [reply content/en/docs/tasks/administer-cluster/out-of-resource.md:124]: is the best way to do it to add a link to the [line 34](https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/administer-cluster/out-of-resource.md#user-content-eviction-signals:~:text=node.stats.rlimit.maxpid) of this file. I agree that table entry may not be descriptive enough to deduct which percentage is used.

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/out-of-resource.md:124]: (BTW This isn't feedback that'd block a merge - we could leave out the emphasis I'm suggesting)

A hyperlink back to the earlier detail sounds reasonable.

**Comment by reylejano** [comment]
@Cweiping kubernetes/kubernetes#98207 has not merged for 1.21
I'm closing this PR and you can re-open this PR and target the `master` branch when [98207](https://github.com/kubernetes/kubernetes/pull/98207) merges or a future `dev-1.xx` branch
/close

**Pull request closed without merging**
